src/api.js

```diff
@@ -4,9 +4,15 @@ const gamesAPI = axios.create({
   baseURL: "https://kanes-games.onrender.com/api",
 });
 
-export const getReviews = async (category) => {
+export const getReviews = async (query) => {
   let path = "/reviews";
-  if (category) path += `?category=${category}`;
+  if (query.category && query.filterName) {
+    path += `?category=${query.category}&sort_by=${query.filterName}&order=${query.filterOrder}`;
```

Comment:
you might look into using the axios `params` option as part of a refactor (and to make your life a bit easier in future)

Reply:
I'll definitely look into this, thanks!